conda_forge_tick/migrators/migration_yaml.py

```diff
@@ -19,6 +21,37 @@
     )
 
 
+def merge_migrator_cbc(migrator_yaml: str, conda_build_config_yaml: str):
+    """Merge a migrator_yaml with the conda_build_config_yaml"""
+    migrator_keys = defaultdict(list)
+    current_key = None
+    for line in migrator_yaml.split("\n"):
+        if not line or line.isspace():
+            continue
+        if re.match(r"\w", line) or line.startswith("_"):
+            current_key = line.split()[0]
+        # fix for bad indentation from bot's PRs
+        if line.startswith('-'):
+            line = '  ' + line
+        migrator_keys[current_key].append(line)
+
+    outbound_cbc = []
+    current_cbc_key = None
+    for line in conda_build_config_yaml.split("\n"):
+        # if we start with a word/underscore we are starting a section
+        if re.match(r'\w', line) or line.startswith('_') or line.startswith('#'):
```

Comment:
Why are we processing comment lines here with the `#`?

Reply:
They often have no spaces before them so they are at the same indentation, if we didn't handle them then they would not be in the resulting yaml.